scripts/run_intergration_test.sh

```
@@ -23,36 +23,55 @@ FILE_COVERAGE_INFO="it_coverage.txt"
 BASEDIR=$(dirname "$0")
 source $BASEDIR/setenv.sh
 
+set -e
+
+echo "mode: atomic" > ${FILE_COVERAGE_INFO}
+echo "MILVUS_WORK_DIR: $MILVUS_WORK_DIR"
+
 TEST_CMD=$@
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)